tests/security/TopicWithoutPermissions/TopicWithoutPermissions.mpc

```diff
@@ -0,0 +1,6 @@
+project (Security*): dcpsexe, dcps_test {
+  requires += no_opendds_safety_profile
+  avoids += no_opendds_security
+  exename = *
+}
```

Comment:
You'll want to add a project to generate the IDL TypeSupport files here. Take a look at the top project in tests/security/attributes/SecurityAttributesTest.mpc